api/routes/api_news.py

```diff
@@ -73,3 +82,54 @@ def specific_article(service: NewsService = Provide[Container.news_service]):
         logger.error(f"Error in POST /: {str(e)}")
         get_custom_error_response(e)
 
+
+
+@news_app.route("/get/recommendation", methods=["GET"])
+@jwt_required()
+@inject
+@handle_exceptions
+@logger.log_function_call()
+async def get_recommendations_endpoint(
+        service_recs: RecommendationService = Provide[Container.recommendation_service],
+        service_news: NewsService = Provide[Container.news_service],
+    ):
+    try:
+        user_email = get_jwt_identity()
+        rec = await service_recs.get_recommendations_from_db(user_email)
```

Comment:
```suggestion
        rec = await service_recs.get_user_recommendations(user_email)
```

no need to specify the data source unless there is a good reason to do so.

just KISS, YAGNI and so on (Google search time)